applications/mssql-server.md

```diff
@@ -3,6 +3,8 @@ layout: page
 title: "SQL Server with Persistent Storage"
 keywords: portworx, container, microsoft, sqlserver, persistent storage, docker, kubernetes
 sidebar: home_sidebar
+redirect_from:
+  - /run_portworx_on_ddc.html
```

Comment:
would this be a better page for the redirect? 
https://docs.portworx.com/applications/docker-registry.html